monai/utils/misc.py

```diff
@@ -84,6 +99,38 @@ def ensure_tuple_rep(tup, dim):
     raise ValueError(f"sequence must have length {dim}, got length {len(tup)}.")
 
 
+def adaptive_size(win_size, img_size):
+    """
+    Adapt `win_size` to `img_size`. Typically used when `win_size` is provided by the user,
```

Comment:
I feel the name `adaptive_size` and the doc-string "Adapt `win_size` to `img_size`." is confusing. Actually it selects size from `win_size` and `img_size`, instead of "adapting" one to the other.
What do you think?